build/bin/release

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+
+chart=$1
```

Comment:
Is this intended to be manually run, or via CI? 

If via CI, how will the script determine which changed charts to build?

Reply:
I assume this can be handled by a separate script that's run by CI, checking the diffs and calling this script n times with the name of the chart to release.